ui/src/translations/fr.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Bienvenu",
+  "name": "Nom",
+  "cpu_capacity": "CPU Capacité",
+  "memory": "Mémoire",
+  "pods_number": "Nombre de pods",
+  "username": "Nom d'utilisateur",
+  "password": "Mot de passe",
+  "submit": "Se connecter",
+  "about": "À propos",
+  "nodes": "Nodes",
+  "log_out": "Se Déconnecter"
```

Comment:
"Se déconnecter" (no uppercase D, like "Se connecter" (no uppercase C)), or simply" Déconnexion" (but then, "Se connecter" should be "Connexion").